tools/system_libs.py

```diff
@@ -384,7 +385,7 @@ def build_objects(self):
     objects = []
     cflags = self.get_cflags()
     for src in self.get_files():
-      o = self.in_temp(os.path.basename(src) + '.o')
+      o = self.in_temp(os.path.splitext(os.path.basename(src))[0] + '.o')
```

Comment:
I think this could be `unsuffixed(os.path.basename(src)``

Reply:
Done